[SV] Added `binary_sensor_HassGetState.yaml` + "som" as skip word

slungs opened the pull request.

larsdunemark commented on sentences/sv/_common.yaml:
```diff
@@ -349,7 +349,11 @@ expansion_rules:
   var: "(var | var är | vart | vart är)"
+  vilka: "(vilka | vilken)"
+  någon: "(någon | något | några)"
+  ingen: "(ingen | inga | inget)"
```
Should we use this expansion_rules for something? Feels like it could be used in the new sentences.

slungs replied:
I added in a sentance for vilka and removed the other two, will add in another pr

slungs commented on tests/sv/binary_sensor_HassGetState.yaml:
```diff
@@ -0,0 +1,53 @@
+        domain: binary_sensor
+        device_class: battery
+        state: "on"
+    response: "Inte någon"
```
Not sure how "genetiv" should be handled here. Could we do some logic within the scope of the `which` response?

larsdunemark replied:
I guess it would be possible to have 2 version of the "which" respond depending to sentence.
It might also be that the "which" respond should be simplifyed to "Ingen/Inget"

slungs commented:
The test is failing for code I have not touched in this pr. It is skipped when run locally.

larsdunemark commented:
@slungs I think the problem could be that we added "som" as skip word and it is part of the testcase for timers. "som heter ...."

slungs commented:
> @slungs I think the problem could be that we added "som" as skip word and it is part of the testcase for timers. "som heter ...."

Yeah you are right. I removed it again and will add it back in another PR if used. I think I got it right now.
